specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/bms.json

```diff
@@ -0,0 +1,3599 @@
+      "get":{
+        "tags":[
+          "CrrOperationStatus"
+        ],
+        "operationId":"CrrOperationStatus_Get",
+        "produces":[
+          "application/json"
+        ],
+        "parameters":[
+          {
+            "$ref":"#/parameters/ApiVersion"
+          },
+          {
+            "$ref":"#/parameters/AzureRegion"
+          },
+          {
+            "$ref":"#/parameters/SubscriptionId"
+          },
+          {
+            "name":"operationId",
+            "in":"path",
+            "required":true,
+            "type":"string"
+          }
+        ],
+        "responses":{
+          "200":{
+            "description":"OK",
+            "schema":{
+              "$ref":"#/definitions/OperationStatus"
+            }
+          }
+        },
+        "x-ms-examples":{
+          "Get Operation Status for Cross Region Restore":{
+            "$ref":"./examples/AzureIaasVm/GetCrrOperationStatus_Get.json"
+          }
+        }
+      }
+    },
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupFabrics/{fabricName}/protectionContainers/{containerName}/protectedItems/{protectedItemName}/recoveryPoints/{recoveryPointId}/accessToken":{
+      "post":{
+        "tags":[
+          "RecoveryPoints"
+        ],
+        "summary":"Returns the Access token for communication between BMS and Protection service",
+        "operationId":"RecoveryPoints_GetAccessToken",
+        "produces":[
+          "application/json"
+        ],
+        "parameters":[
+          {
+            "$ref":"#/parameters/ApiVersion"
+          },
+          {
+            "$ref":"#/parameters/VaultName"
+          },
+          {
+            "$ref":"#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref":"#/parameters/SubscriptionId"
+          },
+          {
+            "name":"fabricName",
+            "in":"path",
+            "description":"Fabric name associated with the container.",
+            "required":true,
+            "type":"string"
+          },
+          {
+            "name":"containerName",
+            "in":"path",
+            "description":"Name of the container.",
+            "required":true,
+            "type":"string"
+          },
+          {
+            "name":"protectedItemName",
+            "in":"path",
+            "description":"Name of the Protected Item.",
+            "required":true,
+            "type":"string"
+          },
+          {
+            "name":"recoveryPointId",
+            "in":"path",
+            "description":"Recovery Point Id",
+            "required":true,
+            "type":"string"
+          }
+        ],
+        "responses":{
+          "200":{
+            "description":"OK",
+            "schema":{
+              "$ref":"#/definitions/CrrAccessTokenResource"
+            }
+          },
+          "400":{
```

Comment:
You intend to have an HTTP 400 be interpreted as a successful API call?

Reply:
No, 400 is not a successful API call. Added it for a specific failure: Bad Request. 
Do you mean that all codes listed here are for successful cases?

Reply:
Correct, all status codes in the `responses` block indicate a successful request.

Reply:
Okay. Where can i specify the unsuccessful cases? This API does return a 400 for Bad Request.

Reply:
You don't, that's what the `default` clause is for.  Think of the `response` section as a switch/case, any status code not in the list goes to `default` where the error model is specified.

Reply:
Okay. So, I will remove HTTP 400 for now and as I mentioned in a previous comment, we will take up this task of adding default cases for all existing APIs in a new PR. Please let me know if that works.